README.md

```diff
@@ -38,7 +38,7 @@
 
 `vue-mdc-adapter` is an integration of
 [Material Components](https://material.io/components/web/)
-for [Vue.js](https://vuejs.org) which follows the best practices 
+for [Vue.js](https://vuejs.org) which follows the best practices
```

Comment:
looks to be only style changes from editor. 
Do you want me to use this markdown style ? or is it ok if I keep mine ?